src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -430,7 +430,7 @@ def update_data(self):
             if not lldp_kvs:
                 continue
             try:
-                time_mark = int(lldp_kvs[b'lldp_rem_time_mark'])
+                time_mark = 0
```

Comment:
please add comment for the motivation behind this change so that in future we can go-back and understand

Reply:
Added comment.